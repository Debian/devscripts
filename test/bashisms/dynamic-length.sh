#!/bin/sh

len=1
f=foo

echo "${f:1}" # BASHISM
echo "${f:$len}" # BASHISM
echo "${f:$len$len}" # BASHISM
echo "${f:${len}}" # BASHISM
