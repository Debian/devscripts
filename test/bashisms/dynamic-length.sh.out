possible bashism in bashisms/dynamic-length.sh line 6 (${foo:3[:1]}):
echo "${f:1}" # BASHISM
possible bashism in bashisms/dynamic-length.sh line 7 (${foo:3[:1]}):
echo "${f:$len}" # BASHISM
possible bashism in bashisms/dynamic-length.sh line 8 (${foo:3[:1]}):
echo "${f:$len$len}" # BASHISM
possible bashism in bashisms/dynamic-length.sh line 9 (${foo:3[:1]}):
echo "${f:${len}}" # BASHISM
